keylime/cloud_verifier_tornado.py

```diff
@@ -691,10 +691,10 @@ def put(self):
                 return
 
             if "reactivate" in rest_params:
-                if not isinstance(db_agent, dict):
-                    agent = _from_db_obj(db_agent)
+                if not isinstance(db_agent, dict):  # FIXME: should always be true...
```

Comment:
This `if not isinstance(db_agent, dict)` check can probably go since the conversion is always need and this should just be `agent = cast(Dict[str, Any], _from_db_obj(db_agent))`

Reply:
I removed the isinstance check since it's always the case that it's not a dict.